### PR TITLE
fix graph and fix filters

### PR DESCRIPTION
**Pull request by rajeshcanaris**

### CODE REVIEW

**Review by umar-ashrfy — APPROVED** [review]

File: app/Http/Controllers/SettingsCani.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
```
Comment (Shubham40Bears) [inline app/Http/Controllers/SettingsCani.php]: Rename this file to ReportSettingsController

File: app/Http/Controllers/SlaReportController.php [hunk 2]
```diff
@@ -14,4 +15,18 @@ public function slaReport()
         $tickets = Conversation::with('user', 'conversationCustomField.custom_field', 'conversationCategory','conversationPriority')->get();
         return view('sla/report', compact('tickets'));
     }
+    // public function settings(){
```
Comment (Shubham40Bears) [inline app/Http/Controllers/SlaReportController.php]: Remove unused code

**Comment by rajeshcanaris** [comment]
Its Fix sir

File: Modules/Reports/Resources/views/partials/menu.blade.php [hunk 3]
```diff
@@ -7,6 +7,7 @@
         <li class="{{ \App\Misc\Helper::menuSelectedHtml('reports.sla') }}"><a href="{{ route('reports.sla') }}">{{ __('SLA Report') }}</a></li>
         <li class="{{ \App\Misc\Helper::menuSelectedHtml('reports.conversations') }}"><a href="{{ route('reports.conversations') }}">{{ __('Conversations Report') }}</a></li>
         <li class="{{ \App\Misc\Helper::menuSelectedHtml('reports.productivity') }}"><a href="{{ route('reports.productivity') }}">{{ __('Productivity Report') }}</a></li>
+        <li><a href="/reports/settings">Settings</a></li>
```
Comment (umar-ashrfy) [inline Modules/Reports/Resources/views/partials/menu.blade.php]: Change the Link to be dynamic. Check above links

File: app/Console/Commands/AutoReport.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Settingssla;
```
Comment (umar-ashrfy) [inline app/Console/Commands/AutoReport.php]: Change the class name from Settingssla to SLASetting

File: app/Console/Commands/AutoReport.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+     * @var string
+     */
+    protected $signature = 'canidesk:auto-reporting';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'this command is uesd to send email automatically';
```
Comment (umar-ashrfy) [inline app/Console/Commands/AutoReport.php]: uesd -> used

File: database/migrations/2023_06_15_174122_create_settingsslas_table.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+class CreateSettingsslasTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('settingsslas', function (Blueprint $table) {
```
Comment (umar-ashrfy) [inline database/migrations/2023_06_15_174122_create_settingsslas_table.php]: sla_setting

File: app/Http/Controllers/ReportSettingsController.php [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+use Illuminate\Http\Request;
+use App\Settingssla;
+
+class ReportSettingsController extends Controller
+{
+
+    public function index(){
+        
+      $settings=Settingssla::orderBy('id', 'desc')->first();
```
Comment (umar-ashrfy) [inline app/Http/Controllers/ReportSettingsController.php]: Change the class name from Settingssla to SLASetting

File: app/Console/Commands/AutoReport.php [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+
+        $settings=Settingssla::orderBy('id', 'desc')->first();
```
Comment (umar-ashrfy) [inline app/Console/Commands/AutoReport.php]: Change the class name from Settingssla to SLASetting

File: app/Http/Controllers/ReportSettingsController.php [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+      return view('/sla/settings',compact('settings'));
+      
+    }
+
+    public function addDataSettings(Request $request){
+    $settingslas=new Settingssla;
```
Comment (umar-ashrfy) [inline app/Http/Controllers/ReportSettingsController.php]: $settingslas=new Settingssla;

Refactor
$slaSettings = new SLASetting();

File: resources/views/layouts/app.blade.php [hunk 10]
```diff
@@ -163,7 +165,7 @@
                                                     @endif
 
                                                     @if ($web_notifications_info['notifications']->hasMorePages())
-                                                        <li class="web-notification-more">
+                                                        <li class="Container-flweb-notification-more">
```
Comment (umar-ashrfy) [inline resources/views/layouts/app.blade.php:167]: Have a space in between two css classes
Container-flweb-notification-more. -> container-fl web-notification-more

**Review by umar-ashrfy — CHANGES_REQUESTED** [review]

File: app/Console/Commands/AutoReport.php [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+        // Output the generated PDF to the browser
+        // $dompdf->save('report.pdf');
+
+        $output = $dompdf->output();
+        file_put_contents('report.pdf', $output);
+        $data = array('name'=>"Example");
+        $mail = Mail::send('mail', $data, function($message) {
+        $message->to('rr7049908@gmail.com', 'Tutorials Point')->subject
+            ('hello i am rajesh rathod')->attach('/home/rathod/git/TaskSecond/canidesk/report.pdf');
```
Comment (Shubham40Bears) [inline app/Console/Commands/AutoReport.php:79]: change this to storage path of laravel

File: app/Console/Commands/AutoReport.php [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+        $dompdf->render();
+
+        // Output the generated PDF to the browser
+        // $dompdf->save('report.pdf');
+
+        $output = $dompdf->output();
+        file_put_contents('report.pdf', $output);
+        $data = array('name'=>"Example");
+        $mail = Mail::send('mail', $data, function($message) {
+        $message->to('rr7049908@gmail.com', 'Tutorials Point')->subject
```
Comment (Shubham40Bears) [inline app/Console/Commands/AutoReport.php:78]: this should be dynamic

**Review by Shubham40Bears — CHANGES_REQUESTED** [review]

**Pull request merged**